package.json

```diff
@@ -23,8 +25,11 @@
   "author": "rrainn, Inc.",
   "license": "MIT",
   "dependencies": {
+    "archiver": "^3.0.0",
+    "archiver-promise": "^1.0.0",
```

Comment:
Do we ever use this?

Reply:
I'll remove archiver-promise. I thought I had uninstalled it but maybe not?